### ISSUE TITLE
Opening a dialog like Ctrl + P and quickly typing ignores the first keystrokes

### ISSUE THREAD

**Issue by hannesfrank** [remnoteio/remnote-issues#857]
Keystrokes between the shortcut to open a dialog (like Ctrl + P) and the dialog actually opening are ignored.
This result is this:
![image](https://user-images.githubusercontent.com/4137110/108622969-8d27fd00-743c-11eb-85e9-1b7f2261f7cb.png)

It requires mental resources to wait for the dialog opening before typing and introduces friction to the editing experience. (Similar to why deterministinc navigation in a text editor using easymotion etc. requires less mental effort than holding down an arrow key and timing the release or mashing it 20 times). 

It also happens with the latex dialog when you type just a bunch of variables inline.

(If not already done) the lag until the dialog shows maybe could be reduced by keep a dialog instance alive in the background instead of freshly creating one each time? Otherwise keystrokes must be recorded before the dialog is ready which I assume is harder?

**Comment by mfranzs** [comment]
Are you still experiencing this? I'm having some trouble reproducing.

**Comment by hannesfrank** [comment]
Your PC is just too fast ;) 

![remnote-ctrl-p](https://user-images.githubusercontent.com/4137110/114143882-fbede680-9914-11eb-82cb-212dfbc5ec24.gif)

I actually had a harder time today reproducing it (works 95% of the time). It seems that inputs before the dialog appears are somewhat captured now?

But there is a different phenomenon happening now - or rahter I did not notice it before, so it might either be a new/different issue/regression or that my CPU was maxed from an update this time intensifying the problem. It also applies to the latex editor which makes fast typing a bit annoying. The process is as follows:
- User: shortcut to open dialog (ctrl + P/$$)
- User: type something, say X
- RemNote: dialog opens, inserts X and _then selects X_
- User: continues typing overwriting selected text

This is best visible on the last test in the gif where I paused after the selection becomes visible.

I feel that the lag and therefore this problem intensifies when the opened document becomes larger.
